README.md

```diff
@@ -1,80 +1,212 @@
-  - [x] cl_rest_client/el_rest_client - DESCOPED as facts will do this
-  - [x] cl_node_metrics_info - pure POJO NO BLOCKERS
-  - [x] get render templates to have the magic strings subsituted with real values
-  - [x] confirm that the 0x30000038 value in `static_files/genesis-generation-config/cl/config.yaml.tmpl` is correct - this makes prysm work [DESCOPED]
-  - [x] confirm behavior of artifact_uuid post for loop fix
\ No newline at end of file
+Ethereum Module
+=======================
+
+This is a [Kurtosis Starlark module][starlark-docs] that will:
+
+1. Generate EL & CL genesis information using [this genesis generator](https://github.com/skylenet/ethereum-genesis-generator)
+1. Spin up a network of mining Eth1 clients
+1. Spin up a network of Eth2 Beacon/validator clients
+1. Add [a transaction spammer](https://github.com/kurtosis-tech/tx-fuzz) that will repeatedly send transactions to the network
+1. Launch [a consensus monitor](https://github.com/ralexstokes/ethereum_consensus_monitor) instance attached to the network
+1. Optionally block until the Beacon nodes finalize an epoch (i.e. finalized_epoch > 0)
+
+For much more detailed information about how the merge works in Ethereum testnets, see [this document](https://notes.ethereum.org/@ExXcnR0-SJGthjz1dwkA1A/H1MSKgm3F).
+
+Quickstart
+----------
+
+1. [Install Docker if you haven't done so already][docker-installation]
+1. [Install the Kurtosis CLI, or upgrade it to the latest version if it's already installed][kurtosis-cli-installation]
+1. Ensure your Docker engine is running:
+   ```bash
+   docker image ls
+   ```
+1. Create a file in your home directory `eth2-module-params.json` with the following contents:
+
+   ```yaml
+   global_client_log_level: "info"
+   ```
+
+1. Execute the module, passing in the params from the file:
+   ```bash
+   kurtosis exec --enclave-id eth2 github.com/kurtosis-tech/eth2-module --args "$(cat ~/eth2-module-params.json)"
+   ```
+
+Management
+----------
+
+Kurtosis will create a new enclave to house the services of the Ethereum network. [This page][using-the-cli] contains documentation for managing the created enclave & viewing detailed information about it.
+
+Configuration
+-------------
+
+To configure the module behaviour, you can modify your `eth2-module-params.yaml` file. The full YAML schema that can be passed in is as follows with the defaults ([from here](https://github.com/kurtosis-tech/eth2-module/blob/master/types.proto) provided:
+
+Note: Following an update starting the network post-merge, `erigon`, `nimbus` and `prysm` clients don't work anymore. Fixes are tracked in the following Github issues:
+- Prysm: [#11508][prysm-issue]
+- Nimbus: [#4193][nimbus-issue]
+- Erigon: [#154][erigon-issue]
+
+<details>
+    <summary>Click to show all configuration options</summary>
+
+<!-- Yes, it's weird that none of this is indented but it's intentional - indenting anything inside this "details" expandable will cause it to render weird" -->
+```json
```

Comment:
Are we tracking the ability to do YAML param files? Actually maybe not worth it right now because we're moving away from proto

Reply:
I've almost added the issue but didn't do it. Will do it when we have proto resolution.